src/scripts/db_store.js

```diff
@@ -0,0 +1,27 @@
+import browserID from "browser-id"
```

Comment:
move this logic to API to reduce surface?

Reply:
clarification: proxy this request through the API as the API already is connected to the DB. we don't need to expose the DB to the FE.